MODULE.bazel

```diff
@@ -51,13 +61,25 @@ http_archive = use_repo_rule("@bazel_tools//tools/build_defs/repo:http.bzl", "ht
 # Tarballs and fetched dependencies (default - use in cases when building from precompiled bin and tarballs)
 #############################################################################################################
 
+# this is used for linux x86_64
 http_archive(
     name = "libtorch",
     build_file = "@//third_party/libtorch:BUILD",
     strip_prefix = "libtorch",
     urls = ["https://download.pytorch.org/libtorch/nightly/cu128/libtorch-cxx11-abi-shared-with-deps-latest.zip"],
 )
 
+# in aarch64 platform you can get libtorch via either local or wheel file
+# It is possible to specify a wheel file to use as the libtorch source by providing the URL below and
+# using the build flag `--//toolchains/dep_src:torch="whl"`
+http_archive(
```

Comment:
Since this likely will always be stale lets comment it out but leave the source for people